### PR TITLE
Makes the Ansem implant locked

### PR DESCRIPTION
**Pull request by alien3301**
## About The Pull Request

The pistol nukies start with (including uplink spawned ones) is now implant locked like their other guns.

## Why It's Good For The Game

Every other gun they get is implant locked. The pistol is pretty underwhelming anyway so I don't think it should be an exception.

## Changelog

:cl:
fix: The Ansem (nuke op pistol) is implant locked like other nukie guns
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
should probably make the fisher-ansem below it unrestricted, or give it an unrestricted variant, since it shows up in a traitor bundle

also maybe make sure this doesn't fudge it for deathmatch loadouts? i know there's the nukie loadout on final destination that uses it

**Comment by alien3301** [comment]
Oh my bad

**Comment by alien3301** [comment]
Tell me if there's any more places where this messes with intentionally unlocked ansems

**Comment by axelzonvolt** [comment]
spy antag can get ansems in their reward pool.

**Comment by SmArtKar** [comment]
Induction kit also has one

**Comment by cnleth** [comment]
Balance change imo. The pistols they had before ansem were also always unrestricted, and there is no indication of intent to make these restricted anywhere

**Comment by Loafin34** [comment]
unsoulful.. the whole idea is if a nukie finds a traitor and they want to team up they give the pistol to the traitor. also, its a funny loot to get from a poorly-equipped nukie that died fast enough to not holster their ansem.

**Comment by alien3301** [comment]
> Induction kit also has one

It also comes with the nukie implant that allows you to use syndie pins.

> unsoulful.. the whole idea is if a nukie finds a traitor and they want to team up they give the pistol to the traitor. also, its a funny loot to get from a poorly-equipped nukie that died fast enough to not holster their ansem.

They can buy the induction kit to give the traitor their hud (team identification S) and allow them to use syndie pins

> Balance change imo. The pistols they had before ansem were also always unrestricted, and there is no indication of intent to make these restricted anywhere

Exact same pr for the nuke op syndie revolver tagged as fix no problem. Also it's supposed to be implant locked since literally every other gun they get is. Why SHOULDN'T the ansem be?

> spy antag can get ansems in their reward pool.

Alrighty. Gonna give them the unlocked one

**Comment by jughu** [comment]
something something anti sandbox something hugbox

**Comment by cnleth** [comment]
> Also it's supposed to be implant locked since literally every other gun they get is. Why SHOULDN'T the ansem be?

It's an emergency sidearm for when you run out of ammo, not your main weapon. Most of the restricted guns are powerful primary weapons. You are seeing a pattern that might not exist in the first place.

**Comment by alien3301** [comment]
> > Also it's supposed to be implant locked since literally every other gun they get is. Why SHOULDN'T the ansem be?
> 
> It's an emergency sidearm for when you run out of ammo, not your main weapon. Most of the restricted guns are powerful primary weapons. You are seeing a pattern that might not exist in the first place.

Why make the emergency sidearm worse by letting people instantly shoot you with it when you drop it?

**Comment by alien3301** [comment]
> something something anti sandbox something hugbox

Disarming people isn't sandbox. You can still print regular firing pins at sec to loot nukie guns

**Comment by cnleth** [comment]
> > > Also it's supposed to be implant locked since literally every other gun they get is. Why SHOULDN'T the ansem be?
> > 
> > 
> > It's an emergency sidearm for when you run out of ammo, not your main weapon. Most of the restricted guns are powerful primary weapons. You are seeing a pattern that might not exist in the first place.
> 
> Why make the emergency sidearm worse by letting people instantly shoot you with it when you drop it?

You are not making it "worse" by not changing it. This is how the weapon behaved for years. You are trying to make this weapon fit your definition of "better". This looks a lot like an ided PR.

**Comment by alien3301** [comment]
> > > > Also it's supposed to be implant locked since literally every other gun they get is. Why SHOULDN'T the ansem be?
> > > 
> > > 
> > > It's an emergency sidearm for when you run out of ammo, not your main weapon. Most of the restricted guns are powerful primary weapons. You are seeing a pattern that might not exist in the first place.
> > 
> > 
> > Why make the emergency sidearm worse by letting people instantly shoot you with it when you drop it?
> 
> You are not making it "worse" by not changing it. This is how the weapon behaved for years. You are trying to make this weapon fit your definition of "better". This looks a lot like an ided PR.

I never used the gun as nuke op. I'm just trying to make it consistent with other guns since there is no good reason for specifically the ansem to have a regular pin by default.

**Comment by alien3301** [comment]
Imo if you want to leave out some guns from being implant locked I'd make it the strongest ones (sniper, L6) so they'd have the downside of potentially backfiring if you get disarmed of them.

**Comment by Loafin34** [comment]
> > unsoulful.. the whole idea is if a nukie finds a traitor and they want to team up they give the pistol to the traitor. also, its a funny loot to get from a poorly-equipped nukie that died fast enough to not holster their ansem.
> 
> They can buy the induction kit to give the traitor their hud (team identification S) and allow them to use syndie pins Remember that all nukie guns can be looted roundstart with electronic firing pins from the sec fab and a screwdriver

ansem was made before induction kit. also, it's 20 tc for an S over their forehead and a potential traitor. also, you can't un-pin in the middle of combat. an assistant getting a lucky shove and popping a cap in a nukie is very soulful.

**Comment by alien3301** [comment]
> ansem was made before induction kit

Okay? Now we have the induction kit.

> it's 20 tc for an S over their forehead and a potential traitor

20 tc during war isn't that much plus you'll have to spend less TC on them than on reinforcements since they have their antag abilities. Wdym by "potential traitor"? The induction kit makes anyone who's valid into a full on nukie who will have to work with you unlike a random traitor. Also gives them a pass to do anything with mrp rules.

> also, you can't un-pin in the middle of combat

Yes. That's how it works. If they lose their gun they lose than gun instead of dying on the spot.

> an assistant getting a lucky shove and popping a cap in a nukie is very soulful.

Disarming them of the big guns would also be very cool and soulful but that's not how it works. If you want you could pr removing syndie pins from all their guns and maybe adding them as a uplink item.

**Comment by alien3301** [comment]
> The pistols they had before ansem were also always unrestricted, and there is no indication of intent to make these restricted anywhere

About this. Aren't they only not implant locked because they're a reworked makarov which traitors used to be able to buy?. That's clearly an oversight. Nukie revolvers also were unlocked because they can be bought by traitors and that was fixed.

**Comment by DrAmazing343** [comment]
This doesn't really feel like an oversight, necessarily— or, if it was, it's persisted long enough that it feels more like a feature. I like the idea of the Ansem as a holdout weapon, or in extenuating circumstances, a weapon turned right back on the Nukie. Call me insane, but I fondly recall a time when I got robusted as a Nukie and had my Ansem used to shoot me down. Would that badass moment happen if it was pinned? Nah, hardly.

I think a lot of specialized pinned weapons are eh in general, but more than anything it feels kinda lame here.

**Comment by alien3301** [comment]
> This doesn't really feel like an oversight, necessarily— or, if it was, it's persisted long enough that it feels more like a feature. I like the idea of the Ansem as a holdout weapon, or in extenuating circumstances, a weapon turned right back on the Nukie. Call me insane, but I fondly recall a time when I got robusted as a Nukie and had my Ansem used to shoot me down. Would that badass moment happen if it was pinned? Nah, hardly.
> 
> I think a lot of specialized pinned weapons are eh in general, but more than anything it feels kinda lame here.

This feels like a noob trap to me. All your guns are locked except this one you always have and you're only told in game if you examine it and notice it has a regular pin. This just makes you get fucked over for no reason. The pistol isn't even that good as a last resort and with it being disarmable you're better off just never pulling it out.

If you really like disarming nukies of their guns you can make all of them unlocked but having just the shitty (compared to their other stuff) pistol they start with not have the special pin is weird.

**Comment by jughu** [comment]
nothing is a noob trap. if i disarm a american in real life from his gun i shoot him sounds a skill issue on his part

**Comment by projectkepler-ru** [comment]
Wasn' this,,, literally intended?
like 
I saw on the wiki 7 years ago that the makarov the nukie start with is meant to not have restriction and warns you that it can get used against you

**Comment by alien3301** [comment]
> Wasn' this,,, literally intended? like I saw on the wiki 7 years ago that the makarov the nukie start with is meant to not have restriction and warns you that it can get used against you

The wiki does say that it's not locked but why does this matter? The wiki is supposed to just say how it is whether it's intended or not.

**Comment by DrAmazing343** [comment]
> If you really like disarming nukies of their guns you can make all of them unlocked but having just the shitty (compared to their other stuff) pistol they start with not have the special pin is weird.

Making all of them unlocked would prolly be a bit too much of a change, but like, the Ansem is a holdout weapon. It makes sense to me that it doesn't have a pin because it's your "backup/last resort" weapon instead of a primary. Also, if it falls into crew hands, that's even cooler for the players.

**Comment by alien3301** [comment]
>It makes sense to me that it doesn't have a pin because it's your "backup/last resort" weapon instead of a primary

If anything the last resort gun should be locked. If you get disarmed of your primary gun while you're doing mostly fine otherwise you have a good chance of not dying on the spot but if you're desperate enough to pull out the pistol you're toast if it's disarmed.

>Also, if it falls into crew hands, that's even cooler for the players.

EVERY GUN CAN BE USED BY NON NUKIES IF THEY GET IT

**Comment by SimplySmiley123** [comment]
Why would you lock the Makarov at home behind the syndie firing pin...
It's not powerful to warrant it, it's not a primary weapon either, it's an emergency backup for after you get disarmed of your primary weapon or when you run out of ammo mid-fight.

If a nukie looses their ansem then die to that ansem they dropped instead of whatever made them stop using their primary, then by all means, they deserve that L.

**Comment by necromanceranne** [comment]
This isn't an oversight. When I swapped them to the Ansem, I noticed the lack of a pin lock and I didn't give them a firing pin lock simply because I felt it was useful for both events AND also its like, the absolute minimum reward for killing a nukie successfully and nabbing their gun. I thought it was cool, so I maintained it.

**Comment by Addust** [comment]
it's underwhelming because it's meant to be the "this can go two ways" gun - if a nukie has to draw their pistol on someone, they have already fucked up bad in one way or another, and giving the crew member that pistol not only gives them a chance to actually fucking finish the operative if their laser, egun, or god forbid fun guns like the WT-550 runs out, but also a way to get the upper hand on avenging operatives

tl:dr let people have fun with the gun, if you REALLY think it should be pinned leave a few spare syndie pins in the nukie base and a note about the ansem

**Comment by jughu** [comment]
Woke solution, make everything not pinned nukie implant pinned and sell a pinbox seperately

**Comment by alien3301** [comment]
Alright
